WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -733,8 +742,8 @@ extension EditableOrderViewModel {
                                                             baseAmountForPercentage: feesBaseAmountForPercentage,
                                                             feesTotal: feeLineTotal,
                                                             didSelectSave: saveFeeLineClosure)
-            self.couponLineViewModel = CouponLineDetailsViewModel(isExistingCouponLine: shouldShowCoupon,
-                                                                  code: couponCode,
+            self.addCouponLineViewModel = CouponLineDetailsViewModel(isExistingCouponLine: false,
+                                                                  code: "",
```

Comment:
I might been missing if there's a reason for this, but should we pass the property here? We're still setting it to an empty string if is not given on init

```suggestion
                                                                  code: couponCode,
```

Reply:
Thanks for the comment here. As this is the view model when we add a new coupon, we initialize it with an empty code. But I see your point, and to make it simpler I made the coupon parameter in the init optional with a nil value, so we don't have to set it. Done in a787cb3353eaac46cbb92df21103ac5006db9f3c